### ISSUE TITLE
Make lasso work with stylus / touch inputs

### ISSUE THREAD

**Issue by fspoettel** [flekschas/regl-scatterplot#204]
Right now, the lasso mode is implemented via `mouse` events. This prevents the lasso tool from being compatible with coarser inputs such as stylus or even touch. It would be nice if the lasso tool would support those. Maybe it is enough already to switch to using the generic`pointer` events, I'm not sure - things like long press would have to be tested.

Use case: annotating plots on a tablet with a stylus.

_This is an issue for our company, so I could work on an implementation._

**Comment by flekschas** [comment]
I agree, would be great if lassoing works on all devices. I'm happy to get a PR merged.
